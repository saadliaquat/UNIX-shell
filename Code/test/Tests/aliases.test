alias ls "echo ls"
alias ls
ls
ls
unalias ls
ls
alias ll "ls -la"
alias grep "grep --color=auto"
alias
alias gs "git status"
gs
ll
unalias ll
alias
alias grep
alias hello "echo hello"
hello
pwd
alias
grep '3' ./config.json