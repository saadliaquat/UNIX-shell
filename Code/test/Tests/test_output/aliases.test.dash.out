ls='echo ls'
ls
ls
Tests
config.json
config.json.log
find.log
names.txt
test.py
grep='grep --color=auto'
ll='ls -la'
total 16
drwxrwxrwx 1 root root  512 Oct  3 01:07 .
drwxrwxrwx 1 root root  512 Oct  3 00:03 ..
drwxrwxrwx 1 root root  512 Oct  3 01:07 Tests
-rwxrwxrwx 1 root root  720 Oct  3 00:01 config.json
-rwxrwxrwx 1 root root    0 Oct  2 23:57 config.json.log
-rwxrwxrwx 1 root root   50 Oct  3 00:43 find.log
-rwxrwxrwx 1 root root   52 Oct  3 00:01 names.txt
-rwxrwxrwx 1 root root 9064 Oct  3 00:01 test.py
grep='grep --color=auto'
gs='git status'
grep='grep --color=auto'
hello
/home/os-fall-2023/code/test
grep='grep --color=auto'
hello='echo hello'
gs='git status'
        "easy": 30,
        "advanced": 30
    "default_timeout": 3,
